.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - run: dockerize -wait tcp://localhost:5432 -timeout 1m
       - run: bundle exec rake db:create
       - run: bundle exec rake db:schema:load
-      - run: bundle exec rake db:insert_states db:upsert_vita_partners
```

Comment:
This is the only actually important part of this PR, for what it's worth.

Reply:
! oh, so we were adding them to the database for CI test runs manually. tricky!